test/libs/user.rb

```diff
@@ -53,9 +53,11 @@ class User < ActiveRecord::Base
     },
     length: { within: 3..100 },
     uniqueness: {
-      case_sensitive: false,
-      if: :will_save_change_to_login?
```

Comment:
Why was the `       if: :will_save_change_to_login?` removed?

Reply:
TBH I’m not actually sure how `will_save_change_to_login` is meant to function here, and would welcome a better understanding.

My reasoning in removing it was that it seemed logical, in making the User model case-sensitive, that we would want to go from "case-insensitive if X" (implying sensitive otherwise) to "always sensitive".

Reply:
I'm not certain, but I think it's a performance optimization, eliminating a potentially unncessary query.

Reply:
Okay, I’ve re-added this code!